docs/pages/server-access/getting-started.mdx

```diff
@@ -76,42 +89,63 @@ the rights and access to read.
 
 <ScopedBlock scope={["oss", "enterprise"]}>
 
-Start the Node. Change `tele.example.com` to the address of your Teleport Proxy
+Configure and start the Node. Change `tele.example.com` to the address of your Teleport Proxy
 Service. Assign the `--token` flag to the path where you saved
 `token.file`.
 
 ```code
-# Join cluster
-$ sudo teleport start \
-   --roles=node \
+# Generate the Teleport configuration file defaulted at /etc/teleport.yaml
+$ sudo teleport node configure \
+   --labels=env=example \
    --token=/path/to/token.file \
-   --auth-server=tele.example.com:443
+   --proxy=tele.example.com:443 \
+   -o file
+# Start and join cluster
+$ sudo teleport start
 ```
 
 </ScopedBlock>
 <ScopedBlock scope={["cloud"]}>
 
-Start the Node. Change `mytenant.teleport.sh` to your Teleport Cloud tenant
+Configure and start the Node. Change `mytenant.teleport.sh` to your Teleport Cloud tenant
 address. Assign the `--token` flag to the path where you saved `token.file`.
 
 ```code
-# Join cluster
-$ sudo teleport start \
-   --roles=node \
+# Generate the Teleport configuration file defaulted at /etc/teleport.yaml
```

Comment:
Same suggestion re: "defaulted at"—this sounds a bit strange to me.

Reply:
Updated